api/proto/config/storage.proto

```diff
@@ -193,3 +193,29 @@ message Volume {
   string displayName = 7;  // Optional friendly name echo'ed in info message
   bool clear_text = 8;  // Flag to indicate the volume encryption needed or not
 }
+
+// ZfsDiskState is an operational state of disks as part of ZFS
+enum ZfsDiskState {
```

Comment:
This should be the (desired) configuration and not the operational state.
The operational state will include other things like failed etc which we have in the separate PR on the info/metrics API.